aclk/aclk.c

```diff
@@ -406,17 +408,36 @@ void aclk_graceful_disconnect(mqtt_wss_client client)
     mqtt_wss_disconnect(client, 1000);
 }
 
+static unsigned long aclk_reconnect_delay() {
+    unsigned long recon_delay;
+
+    if (aclk_disable_runtime) {
+        aclk_tbeb_delay(0, 0, 0, 0);
+        return 60 * MSEC_PER_SEC;
+    }
+
+    if (aclk_block_until) {
+        recon_delay = aclk_block_until - now_monotonic_sec();
```

Comment:
Need to check that `aclk_block_until` is GT `now_monotonic_sec()` before computing `recon_delay`.

Reply:
aclk_block_until is set as follows:
`aclk_block_until = now_monotonic_sec() + backoff;`
and used only if set (non 0)
therefore my thinking is due to use of monotonic clock (which is defined as clock that ever only increments) it cannot result to negative value

Reply:
But it will be "negative" if more than `backoff` seconds have passed since setting `aclk_block_until`. Is there a strong guarantee that this can not happen (today and in the future)?

Reply:
well normally it happens right away but will add check as you said in future code can change and this might be forgoten